### ISSUE TITLE
QCAR Folder

### ISSUE THREAD

**Issue by teatherconsulting** [yasirkula/UnityIonicIntegration#29]
When we export our project for iOS there isn't a Data\Raw\QCAR folder.  Instead there is a Data\Raw\Vuforia folder.  Is there something that has to be done to get the QCAR folder?

**Comment by teatherconsulting** [comment]
Sorry, I missed that one!  That was exactly what I needed.  Thank you for your prompt reply!  

I was actually hoping that might solve my problem, but I am still getting the following error:

Exception Type:  EXC_BAD_ACCESS (SIGSEGV)
Exception Subtype: KERN_INVALID_ADDRESS at 0x0000000000000001
VM Region Info: 0x1 is not in any region.  Bytes before following region: 4308156415
      REGION TYPE                      START - END             [ VSIZE] PRT/MAX SHRMOD  REGION DETAIL
      UNUSED SPACE AT START
--->  
      __TEXT                 0000000100c94000-000000010301c000 [ 35.5M] r-x/r-x SM=COW  ...HoopsForHeart

The splash screen shows for a second and then the app crashes completely with the error above.  I did see something similar in #15 and have already included the Other C Flags and Other C++ Flags.  Any other ideas on what it could be?

**Comment by yasirkula** [comment]
Also check out this comment and see if this is also the case for you: https://github.com/yasirkula/UnityIonicIntegration/issues/24#issuecomment-384086231

**Comment by teatherconsulting** [comment]
We did double check the way our Data folder (and Classes and Libraries and Vuforia) were included and they all seem correct.  We'll keep looking into it and see what we can track down.

On a different note, do you think it would be possible to include multiple Unity projects using this plugin?  We have a couple of different places in our app where we would like to open different Unity games/AR experiences that are each different Unity projects.  We were starting with just a single Unity/Vuforia project, but it seems like multiple Unity projects in one x-code project is likely not possible with this plugin.

**Comment by yasirkula** [comment]
I don't think it is possible with this plugin. Even without this plugin, the same Unity source files coming from multiple Unity projects may result in conflicts during build phase.
